internal/services/analytics_data_export/datasource_analytics_data_exports.go

```diff
@@ -0,0 +1,281 @@
+										MarkdownDescription: "Any message associated with the status",
+										Computed:            true,
+									},
+								},
+							},
+						},
+						"sink": schema.SingleNestedAttribute{
+							MarkdownDescription: "The sink configuration for analytics data",
+							Required:            true,
+							Attributes: map[string]schema.Attribute{
+								"id": schema.StringAttribute{
+									MarkdownDescription: "The resource ID of the sink",
+									Computed:            true,
+								},
+								"type": schema.StringAttribute{
+									MarkdownDescription: "The type of the sink",
+									Computed:            true,
+								},
+								"subscription_id": schema.StringAttribute{
+									MarkdownDescription: "The Azure subscription ID",
+									Computed:            true,
+								},
+								"resource_group_name": schema.StringAttribute{
+									MarkdownDescription: "The Azure resource group name",
+									Computed:            true,
+								},
+								"resource_name": schema.StringAttribute{
+									MarkdownDescription: "The name of the sink resource",
+									Computed:            true,
+								},
+								"key": schema.StringAttribute{
+									MarkdownDescription: "The key for accessing the sink",
+									Computed:            true,
+								},
+							},
+						},
+						"package_name": schema.StringAttribute{
+							MarkdownDescription: "The package name for the analytics data",
+							Computed:            true,
+						},
+						"scenarios": schema.ListAttribute{
+							MarkdownDescription: "The list of scenarios covered by this analytics export",
+							Computed:            true,
+							ElementType:         types.StringType,
+						},
+						"resource_provider": schema.StringAttribute{
+							MarkdownDescription: "The resource provider for the analytics data",
+							Computed:            true,
+						},
+						"ai_type": schema.StringAttribute{
+							MarkdownDescription: "The AI type for the analytics data",
+							Computed:            true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func (d *AnalyticsExportDataSource) Configure(ctx context.Context, req datasource.ConfigureRequest, resp *datasource.ConfigureResponse) {
+	ctx, exitContext := helpers.EnterRequestContext(ctx, d.TypeInfo, req)
+	defer exitContext()
+
+	if req.ProviderData == nil {
+		// ProviderData will be null when Configure is called from ValidateConfig. It's ok.
+		return
+	}
+
+	client, ok := req.ProviderData.(*api.ProviderClient)
+	if !ok {
+		resp.Diagnostics.AddError(
+			"Unexpected ProviderData Type",
+			fmt.Sprintf("Expected *api.ProviderClient, got: %T. Please report this issue to the provider developers.", req.ProviderData),
+		)
+		return
+	}
+
+	d.analyticsExportClient = NewAnalyticsExportClient(client.Api, tenant.NewTenantClient(client.Api))
+}
+
+// mapAnalyticsDataDtoToModel converts an AnalyticsDataDto to AnalyticsDataExportModel.
+func mapAnalyticsDataDtoToModel(dto *AnalyticsDataDto) *AnalyticsDataModel {
```

Comment:
change the mapAnalyticsDataDtoToModel function name to "convertDtoToModel" and move it to models.go file